RxSwift/Subjects/Variable.swift

```diff
@@ -10,7 +10,7 @@
 ///
 /// Unlike `BehaviorSubject` it can't terminate with error, and when variable is deallocated
 /// it will complete it's observable sequence (`asObservable`).
-public final class Variable<Element> {
+public final class Variable<Element>: ObservableConvertibleType {
```

Comment:
(Just for memo) `Variable` should not conform to `ObserverType` because terminal events will be ignored, i.e. `variable.on(.completed))` then `variable.on(.next(value))` will send value. (And using `fatalError` inside is not an option...)